content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -182,6 +182,10 @@ echo $(kubectl get pods --selector=$sel --output=jsonpath={.items..metadata.name
 # Also uses "jq"
 for item in $( kubectl get pod --output=name); do printf "Labels for %s\n" "$item" | grep --color -E '[^/]+$' && kubectl get "$item" --output=json | jq -r -S '.metadata.labels | to_entries | .[] | " \(.key)=\(.value)"' 2>/dev/null; printf "\n"; done
 
+OR
```

Comment:
@dpandhi-git This "OR" can be put in the comment

Reply:
Thank you for the comments @tengqm @hpandeycodeit